app/assets/sass/patterns/_step-by-step-nav.scss

```diff
@@ -271,7 +277,10 @@ $top-border: solid 2px govuk-colour("mid-grey", $legacy: "grey-3");
     }
 
     .app-step-nav__toggle-link {
+      @include govuk-link-hover-decoration;
+      color: $govuk-link-hover-colour;
       text-decoration: underline;
+      text-underline-offset: $govuk-link-underline-offset;
```

Comment:
I'm a little confused by this; reading the code I thought it would mean the thicker underlines would be used, however that's not what I'm seeing currently in Chrome, in the hover state the link 'Show' with class `.app-step-nav__toggle-link` is underlined with a 1px thick underline:

<img width="392" alt="Hover state of step in step by step navigation in Chrome." src="https://user-images.githubusercontent.com/503614/117264836-86a9fe80-ae4b-11eb-87b0-012eee6f9a8f.png">

In Safari I see what I expect:

<img width="351" alt="Hover state of step in step by step navigation in Safari." src="https://user-images.githubusercontent.com/503614/117265706-6af32800-ae4c-11eb-89a8-bfbfd3cc703d.png">

I think what is happening (based on a recollection of something @36degrees said) is that `text-decoration: underline` is setting the default underline thickness in Chrome but not Safari.

So I guess the question is, for the step by step toggle links, which is the behaviour we want?

Reply:
I'm not sure if we can because we want the hover state to apply when the parent button is hovered...but I might be missing something! I have tweaked this to use the following mixins instead of hard-coding some of these styles now though:

```
@include govuk-link-common;
@include govuk-link-hover-decoration;
```

Reply:
Nice! 🙌